src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -20,6 +20,16 @@ image_pyramid:
     description: Image Pyramid
 IMC:
     description: Imaging Mass Cytometry
+
+lc-ms_label-free:
+    description: Label-free LC-MS
+lc-ms_labeled:
+    description: Labeled LC-MS
+lc-ms-ms_label-free:
+    description: Label-free LC-MS/MS
+lc-ms-ms_labeled:
+    description: Labeled LC-MS/MS
+

```

Comment:
These look good and cover all the MS assay types we currently have. These should replace the existing assay type labels for MS when practical.